src/content/developers/docs/consensus-mechanisms/pos/keys/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Keys in proof-of-stake Ethereum
+description: An explanation of keys used in Ethereum since the transition to proof-of-stake.
+lang: en
+sidebar: true
+---
+
+Ethereum secures user's assets using public-private key cryptography. The public key is used as the basis for an Ethereum address - that is, it is visible to the general public and used as a unique identifier. The private (or 'secret') key should only ever be accessible to an account owner. The private key is used to 'sign' transactions and data so that cryptography can prove that the holder approves some action of a specific private key.
+
+Ethereum's keys are generated using [elliptic-curve cryptography](https://en.wikipedia.org/wiki/Elliptic-curve_cryptography).
+
+However, when Ethereum switched from [proof-of-work](/developers/docs/consensus-mechanisms/pow) to [proof-of-stake](/developers/docs/consensus-mechanisms/pos) a new type of key was added to Ethereum. The original keys still work exactly the same as before - there were no changes to the elliptic-curve-based keys securing accounts. However, users needed a new type of key for participating in proof-of-stake by staking ETH and running validators. This need arose from scalability challenges associated with many messages passing between large numbers of validators that required a cryptographic method that could easily be aggregated to reduce the amount of communication required for the network to come to consensus.
+
+This new type of key uses the [Boneh-Lyn-Shacham (BLS) signature scheme](https://wikipedia.org/wiki/BLS_digital_signature). BLS enables a very efficient aggregation of signatures but also allows reverse engineering of aggregated individual validator keys and is ideal for managing actions between validators.
+
+## The two types of validator key {#two-types-of-key}
+
+Before the switch to proof-of-stake, Ethereum users only had a single elliptic-curve-based private key to access their funds. With the introduction of proof-of-stake, users that wished to be solo stakers also required a **validator key** and a **withdrawal key**.
+
+### The validator key {#validator-key}
+
+The validator signing key consists of two elements:
+
+- Validator **private** key
+- Validator **public** key
+
+The purpose of the validator private key is to actively sign on-chain operations such as block proposals and attestations. Therefore these keys have to be held in a hot wallet.
+
+This flexibility has the advantage to move validator signing keys very quickly from one device to another, however, if they have gotten lost or stolen, a thief may be able to **act maliciously** in the following ways:
```

Comment:
```suggestion
This flexibility has the advantage of moving validator signing keys very quickly from one device to another. However, if they have gotten lost or stolen, a thief may be able to **act maliciously** in several ways:
```

Reply:
> English question, wouldn't wrapping "however" in commas be the more appropriate way to use it here? Always tried to avoid using it to start sentences where possible.

I changed it to a new sentence because the use of 'however' created a [run-on sentence](https://www.grammarly.com/blog/run-on-sentence-basics/). Using a semi-colon instead would also be correct.

re: your aversion to beginning sentences with 'However', I've not really heard of that before 😛 Maybe a British/American English difference?

> Also, not sure I'd switch to "several" here... It's really just two things, they could get you slashed or they could exit the validator.

I'm good with using 'in a few ways' here. My edit was to remove the implication that the list was a complete list of potentially malicious activities.